frontend/package.json

```diff
@@ -37,12 +38,13 @@
   "scripts": {
     "start": "vite --port 3000 --host",
     "start:local": "VITE_BACKEND_API_BASE_URL=https://circlesapi.csesoc.app VITE_ENV=dev vite --port 3000 --host",
-    "build": "vite build",
+    "build": "export NODE_OPTIONS=--max_old_space_size=4096; vite build",
```

Comment:
reminder for @gtangelo, check if this should be include/or if there is another way around it

Reply:
Yea I've seen this before, we should make another card to remove it. Also, we might crash Wheatley, ask @lhjt

Reply:
No code is built on Wheatley, so you don't have to worry about resource consumption at build time 😄